### PR TITLE
feat(theme-translations): add extra Korean translation, fix typo

### PR DESCRIPTION
**Pull request by revi**
## Motivation

Updated a few translation string that was left untranslated (probably added after the initial translation?). Those left are because I had no idea where it belongs to. Also fixing a typo with Version.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

This probably does not require test plans.

## Related PRs

N/A

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Hi, thanks for your time on this! #5849 has done some refactorings to the translation system, which will cause merge conflicts in your PR after that one got merged (or vice versa: we merge yours and have to fix the conflicts in that one.) We'd also like to try out the new translation workflow. So while your PR looks good to me (especially the fact that it fixes the "ko only has one plural form" warning), would you kindly stay with us for a while and resolve the conflicts after we merge #5849?

File: packages/docusaurus-theme-classic/codeTranslations/ko.json [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
   "theme.SearchPage.documentsFound.plurals": "{count}개 문서를 찾았습니다.",
   "theme.SearchPage.emptyResultsTitle": "문서를 검색합니다.",
   "theme.SearchPage.existingResultsTitle": "\"{query}\"개 검색 결과가 있습니다.",
   "theme.SearchPage.fetchingNewResults": "새로운 검색 결과를 얻을 수 있습니다...",
   "theme.SearchPage.inputLabel": "검색",
   "theme.SearchPage.inputPlaceholder": "여기에 검색할 키워드를 입력하세요.",
   "theme.SearchPage.noResultsText": "검색 결과가 없습니다.",
-  "theme.TOCCollapsible.toggleButtonLabel": "On this page",
+  "theme.TOCCollapsible.toggleButtonLabel": "이 문서에서",
   "theme.blog.archive.description": "Archive",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:27]: "Archive" is just "blog archive", as for this page: https://docusaurus.io/blog/archive

I don't know if just 아카이브 makes sense

File: packages/docusaurus-theme-classic/codeTranslations/ko.json [hunk 2]
```diff
@@ -48,16 +48,16 @@
   "theme.docs.sidebar.expandButtonAriaLabel": "사이드바 열기",
   "theme.docs.sidebar.expandButtonTitle": "사이드바 열기",
   "theme.docs.tagDocListPageTitle": "{nDocsTagged} with \"{tagName}\"",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:50]: This is for this page: https://docusaurus.io/tests/docs/tags/a something like "2 docs tagged with "a"". "2 docs tagged" is already translated below so you would have to consider how to splice that with "with "a"" while making it make sense.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json [hunk 3]
```diff
@@ -48,16 +48,16 @@
...
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "{count} 개 문서가 태그되었습니다",
   "theme.docs.versions.latestVersionLinkLabel": "최신 버전",
   "theme.docs.versions.latestVersionSuggestionLabel": "최신 문서는 {latestVersionLink} ({versionLabel})을 확인하세요.",
-  "theme.docs.versions.unmaintainedVersionLabel": "{siteTitle} {versionLabel} 문서는 업데이트되지 않습니다.",
+  "theme.docs.versions.unmaintainedVersionLabel": "{siteTitle} {versionLabel} 문서는 더 이상 업데이트되지 않습니다.",
   "theme.docs.versions.unreleasedVersionLabel": "{siteTitle} {versionLabel} 문서는 아직 정식 공개되지 않았습니다.",
   "theme.lastUpdated.atDate": " {date}에",
   "theme.lastUpdated.byUser": " {user}가",
   "theme.lastUpdated.lastUpdatedAtBy": "{atDate}{byUser} 마지막으로 업데이트했습니다.",
   "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← Back to main menu",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/codeTranslations/ko.json:59]: This is the button you see on a mobile when in a doc page:

image

Honestly a naive translation like 메인 메뉴로 돌아가기 would work?

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Not experienced with Korean at all but hope these comments can help you complete the translations

**Comment by revi** [comment]
> Hi, thanks for your time on this! #5849 has done some refactorings to the translation system, which will cause merge conflicts in your PR after that one got merged (or vice versa: we merge yours and have to fix the conflicts in that one.) We'd also like to try out the new translation workflow. So while your PR looks good to me (especially the fact that it fixes the "ko only has one plural form" warning), would you kindly stay with us for a while and resolve the conflicts after we merge #5849?

This one can wait. Just ping me when the 5849 is merged and this can be worked on.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json (hunk 1 above)
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: I don't think so, added as "List of blog articles" based on original translator's work below.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json (hunk 2 above)
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: Gotcha, added.

File: packages/docusaurus-theme-classic/codeTranslations/ko.json (hunk 3 above)
Comment (revi) [reply packages/docusaurus-theme-classic/codeTranslations/ko.json]: It does. Added.

**Review by Josh-Cena — APPROVED** [review]
Thanks 👍

"Blog archive" is not easy to translate. I went for "blog history" in `zh-Hans`, but "blog list" also sounds good

**Comment by slorber** [comment]
Thanks

Theme changes have been merged, it may be simpler to create another new PR with these changes ported to the new system:

https://github.com/facebook/docusaurus/tree/main/packages/docusaurus-theme-translations

**Comment by Josh-Cena** [comment]
At least I wouldn't like seeing a PR closed on my timeline😄 @revi Whether to submit a new one or just resolve conflicts in the current one (or let me do it, if you wish), your call

**Comment by revi** [comment]
@Josh-Cena You can feel free to rebase and merge :-) (Thanks in advance!)

**Comment by Josh-Cena** [comment]
Ah, okay, I'll do it then. Thanks again!

**Comment by Josh-Cena** [comment]
@revi actually, I do not have write access because your fork is an org fork not a personal fork. Do you see a checkbox on the right sidebar to allow write access to maintainers?

**Pull request merged**